src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/EntitySubjectsWidget.java

```diff
@@ -0,0 +1,102 @@
+package org.sagebionetworks.web.client.widget.accessrequirements;
+
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+import java.util.ArrayList;
+import java.util.List;
+import org.sagebionetworks.repo.model.Reference;
+import org.sagebionetworks.repo.model.RestrictableObjectDescriptor;
+import org.sagebionetworks.repo.model.RestrictableObjectType;
+import org.sagebionetworks.repo.model.request.ReferenceList;
+import org.sagebionetworks.web.client.PortalGinInjector;
+import org.sagebionetworks.web.client.utils.CallbackP;
+import org.sagebionetworks.web.client.widget.asynch.IsACTMemberAsyncHandler;
+
+public class EntitySubjectsWidget
+  implements EntitySubjectsWidgetView.Presenter, IsWidget {
+
+  EntitySubjectsWidgetView view;
+  IsACTMemberAsyncHandler isACTMemberAsyncHandler;
+
+  CallbackP<List<RestrictableObjectDescriptor>> onUpdate;
+  String currentEntityIDsValue = "";
+
+  @Inject
+  public EntitySubjectsWidget(
+    EntitySubjectsWidgetView view,
+    PortalGinInjector ginInjector,
+    IsACTMemberAsyncHandler isACTMemberAsyncHandler
+  ) {
+    this.view = view;
+    this.isACTMemberAsyncHandler = isACTMemberAsyncHandler;
+    view.setVisible(false);
+    view.setPresenter(this);
+  }
+
+  @Override
+  public Widget asWidget() {
+    return view.asWidget();
+  }
+
+  public void configure(final List<RestrictableObjectDescriptor> subjects) {
+    configure(subjects, null);
+  }
+
+  public void configure(
+    final List<RestrictableObjectDescriptor> subjects,
+    CallbackP<List<RestrictableObjectDescriptor>> onUpdate
+  ) {
+    this.onUpdate = onUpdate;
+    isACTMemberAsyncHandler.isACTActionAvailable(
+      new CallbackP<Boolean>() {
+        @Override
+        public void invoke(Boolean isACT) {
+          view.setVisible(isACT);
```

Comment:
Why only show subjects for ACT? This could be useful for data administrators, Sage employees, etc

Reply:
This is a good question.  Generally speaking, I don't think most users will ever see the old AR renderers (because the link to the AR page is an ACT-only command).  Since it wasn't shown before, I'm not going to change it in this PR.  But happy to revisit